class FahException(Exception):
    pass

class AuthException(FahException):
    pass
